recipe/meta.yaml

```diff
@@ -43,6 +43,7 @@ requirements:
     - xorg-libxt
     - xz
     - zlib
+    - glib
```

Comment:
Should it be in run or no?

Reply:
run exports should deal with this right?

Reply:
all the way back from 2.54.3 it had the runexports

https://github.com/conda-forge/glib-feedstock/commit/8a533854674a9bb51f734e5c59f32a9e4678a914#